orchagent/portsorch.cpp

```diff
@@ -1853,7 +1853,7 @@ bool PortsOrch::initPort(const string &alias, const int index, const set<int> &l
 
                 m_portList[alias].m_init = true;
 
-                SWSS_LOG_ERROR("Initialized port %s", alias.c_str());
+                SWSS_LOG_INFO("Initialized port %s", alias.c_str());
```

Comment:
Thanks Mahesh. I had this changed in one of my open PR. Could you also change one more log?

https://github.com/Azure/sonic-swss/pull/1497/files#diff-bb736c6a11da68afacf95d761fb52a3c11726cdd72c967d4803b8c9b1a6dcb69L1786